src/common/mmo.h

```diff
@@ -196,6 +196,10 @@
 #error FIXED_INVENTORY_SIZE must be same or smaller than MAX_INVENTORY
 #endif
 
+#if PACKETVER >= 20131223
+	#define OFFICIAL_GUILD_STORAGE
```

Comment:
It might be worth making it easier to opt out of this via -D cflags instead of having it controlled by packetver only (since there's nothing in recent packetvers that prevents using the old system that doesn't require skill points, if one wishes to)